definitions/variable/grids infrastructure/gas_grid.yaml

```diff
@@ -0,0 +1,57 @@
+- Capacity|Natural Gas|Regasifier:
+        description: Regasifier upperbound capacity
+        unit: GW
+- Capacity|Hydrogen|Regasifier:
+        description: Regasifier upperbound capacity
+        unit: GW
+- Capacity|Natural Gas|Storage|Aquifer:
+        description: Aquifer storage capacity
+        unit: GWh
+- Capacity|Natural Gas|Storage|Salt Cavern:
+        description: Salt Cavern storage capacity
+        unit: GWh
+- Capacity|Natural Gas|Storage|Depleted Field:
+        description: Depleted Field storage capacity
+        unit: GWh
+- Injection Rate|Natural Gas|Storage|Aquifer:
+        description: Aquifer storage Injection Rate
+        unit: GW
+- Injection Rate|Gas|Storage|Salt Cavern:
+        description: Salt Cavern storage Injection Rate
+        unit: GW
+- Injection Rate|Gas|Storage|Depleted Field:
+        description: Depleted Field storage Injection Rate
+        unit: GW
+- Extraction Rate|Gas|Storage|Aquifer:
+        description: Aquifer storage Extraction Rate
+        unit: GW
+- Extraction Rate|Gas|Storage|Salt Cavern:
+        description: Salt Cavern storage Extraction Rate
+        unit: GW
+- Extraction Rate|Gas|Storage|Depleted Field:
+        description: Depleted Field storage Extraction Rate
+        unit: GW
+- Network|Gas|Natural Gas|Capacity:
```

Comment:
Assuming that methane and hydrogen cannot use the same networks, I suggest to simplify to 
- Network|Natural Gas|Capacity
- Network|Hydrogen|Capacity